tests/test_masks.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from src.masks import get_mask_account
+from src.masks import get_mask_card_number
+
+
+def test_get_mask_account(number_of_account_for_masks: str) -> None:
+    # Проверка на правильность маскировки счета
```

Comment:
Комментарии к тестам правильно что делаешь, только пиши их в тройных кавычках (как докстринг к функции)
```suggestion
    """Проверка на правильность маскировки счета"""
```

Reply:
исправил